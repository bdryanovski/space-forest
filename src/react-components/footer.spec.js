import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './footer'

describe('Footer', () => {
  it('should match default snapshot', () => {
    const component = renderer.create(<Footer>Hello</Footer>)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
