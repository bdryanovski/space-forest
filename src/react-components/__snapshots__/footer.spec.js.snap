// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match default snapshot 1`] = `
<footer>
  <center>
    Hello
  </center>
</footer>
`;
